src/utils/constants.ts

```diff
@@ -1 +1,13 @@
-export const SummarySectionCode = 'SummarySection'
\ No newline at end of file
+const BUTTON_NEXT = 'Next'
+const BUTTON_PREVIOUS = 'Previous'
+const BUTTON_SUBMIT = 'Submit application'
+const TITLE_COMPANY_PLACEHOLDER = 'ABC Company'
+const TITLE_PROGRESS = 'Introduction'
+
+export {
+    BUTTON_NEXT,
+    BUTTON_PREVIOUS,
+    BUTTON_SUBMIT,
+    TITLE_COMPANY_PLACEHOLDER,
+    TITLE_PROGRESS
+}
```

Comment:
Do you think we want to have to keep track of a bunch of individual imports/exports for different strings? Wouldn't it be easier to just have one `export/import strings` with all the strings and then just putting them in place as `strings.BUTTON_NEXT`, etc.?

Reply:
Ok. I'm convinced now hahah 2 x 1. I'll change to be like that : )

Reply:
If you think it makes it harder to read in code just rename it to a single character on import.
`import { strings as 's' } from 'contants'`

Then can use `s.BUTTON_NEXT` ? (Or `str`, `c`, `_` -- whatever you fancy)